cinema/models.py

```diff
@@ -135,7 +138,8 @@ def save(
 
     def __str__(self):
         return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
+            f"{str(self.movie_session)} "
+            f"(row: {self.row}, seat: {self.seat})"
```

Comment:
The `__str__` method for the `Ticket` model is correctly implemented, but ensure that the `movie_session` object has a meaningful `__str__` method to provide useful information when converting the ticket to a string.